packages/elements/src/components/ui/spinner/Spinner.module.css

```diff
@@ -1,31 +1,34 @@
 .spinner {
-  animation: rotator 1.4s linear infinite;
+  animation: rotator 2s linear infinite both;
 
   circle {
-    stroke-dasharray: 187;
-    stroke-dashoffset: 0;
-    stroke-width: 6;
     stroke-linecap: round;
     transform-origin: center;
+    fill: transparent;
+    stroke-dasharray: 283;
+    stroke-dashoffset: 280;
+    stroke-width: 10px;
```

Comment:
Is this part competing with `stroke-width="6"` in spinner-large.svg? Could we remove one of them?